examples/Spirit/spirit_utils.h

```diff
@@ -91,9 +97,27 @@ std::tuple<  std::vector<std::unique_ptr<dairlib::systems::trajectory_optimizati
           std::vector<int> knotpointVect, // Matrix of knot points for each mode  
           std::vector<Eigen::Vector3d> normals,
           std::vector<Eigen::Vector3d> offsets, 
-          std::vector<double> mus );
+          std::vector<double> mus, 
+          std::vector<double> minTs, 
+          std::vector<double> maxTs );
 
+/// Overload allows the use of the ModeSequenceHelper Class so we dont need to feed in all the arguments
+template <typename T>
+std::tuple<  std::vector<std::unique_ptr<dairlib::systems::trajectory_optimization::DirconMode<T>>>,
+             std::vector<std::unique_ptr<multibody::WorldPointEvaluator<T>>> ,
+             std::vector<std::unique_ptr<multibody::KinematicEvaluatorSet<T>>>   
+          >     
+    createSpiritModeSequence( 
+          drake::multibody::MultibodyPlant<T>& plant, // multibodyPlant
+          dairlib::ModeSequenceHelper* msh );
 
+/// This overload sets all the joints to their nominal limit's
+///    @param plant a pointer to a multibodyPlant
+///    @param trajopt a ponter to a Dircon<T> object  
+template <typename T>
```

Comment:
Is this code actually new? It seems like something we already had,

Reply:
Not new moved in the overload order